src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+
+.stopwatch {
+  position: relative;
+
+  &__border {
+    width: 80vmin;
+    height: 80vmin;
+    border-radius: 50%;
+    border: 1vmin dotted #000;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), hexadecimal color codes should be written in lowercase. Please change `#000` to `#000`.

Reply:
XD